tests/MigrationTest.php

```diff
@@ -64,18 +64,18 @@ public function testCreateMethod()
 
             $this->assertTrue(true, 'Migration ok');
         } catch (\Exception $e) {
-            $this->assertTrue(false, $e->getMessage());
+            $this->assertFalse(false, $e->getMessage());
         }
 
         try {
             $this->migration->alter('bow_tests', function (SQLGenerator $generator) {
                 $generator->dropColumn('name');
                 $generator->addColumn('age', 'int', ['size' => 11, 'default' => 12]);
             });
-            
+
              $this->assertTrue(true, 'Migration ok');
         } catch (\Exception $e) {
-            $this->assertTrue(false, $e->getMessage());
+            $this->assertFalse(false, $e->getMessage());
```

Comment:
It has the same reason I mention on previous comment why I change into `assertFalse` assertion.

Reply:
Nice, @peter279k.